docs/operations/.pages

```diff
@@ -0,0 +1,17 @@
+nav:
+  - installation.md
+  - updating_and_deletion.md
+  - api_validation.md
+  - virtctl_client_tool.md
+  - live_migration.md
+  - hugepages.md
+  - component_monitoring.md
+  - authorization.md
+  - annotations_and_labels.md
+  - node_assignment.md
+  - node_maintenance.md
+  - node_overcommit.md
+  - unresponsive_nodes.md
+  - vmctl.md
+  - basic_use.md
```

Comment:
I believe that the basic usage page should be towards the top of the list, probably after installation

Reply:
Fixed.